packages/server/src/integrations/tests/utils/images.ts

```diff
@@ -0,0 +1,17 @@
+import dotenv from "dotenv"
+import { join } from "path"
+
+const path = join(__dirname, "..", "..", "..", "..", "datasource-sha.env")
+dotenv.config({
+  path,
+})
+
+export function getImageSHAs() {
```

Comment:
Out of curiosity, why is this a function instead just exporting the strings?

Reply:
good point!